### ISSUE TITLE
Disable looking in folders on remote ?

### ISSUE THREAD

**Issue by amrography** [lukasz-wronski/vscode-ftp-sync#204]
Hi,

I want to disable looking inside folders on remote when I run `> Ftp-synce : local to remote`, because it takes too much time.

I tried to add configure my my **ftp-sync.json** but didn't work:
```
"ignore": [
        "\\.vscode",
        "\\.git",
        "\\.DS_Store",
        "\\Folder_Name"
   ]
```

**Comment by MatthewGieger** [comment]
@amrography  I'm also having a similar issue, but you could try removing the "//" from the beginning of your folder names as long as those folders are in the root directory on the ftp server.

**Comment by Tatranskymedved** [comment]
@meichthys  I tried it without "\\" at the start, however that doesn't resolve the case. I guess the plugin should is not looking into the paths that should be ignored for remote location. This should be added as a functionality.

**Comment by meichthys** [comment]
@Tatranskymedved  I agree. We either need more documentation on this, or updated functionality.

**Comment by meichthys** [comment]
@amrography This issue is stale. We should probably close it.
